src/tests/unittests/measurements/test_scans.py

```diff
@@ -110,8 +111,8 @@ def test_scanjob_record_label(self):
         dataset = scan1D(station, scanjob, liveplotwindow=False, verbose=0)
         self.assertTrue(dataset.location.endswith(record_label))
 
-    @staticmethod
-    def test_scan2D(verbose=0):
+    def test_scan2D(self):
+        verbose = 0
```

Comment:
The `verbose=0` argument was used to help using and debugging the test functions. If it is removed as a default argument, we might as well remove it altogether.

Reply:
This was a temporary quick fix, I accidentally left in. I will put it back.